unreleased_history/behavior_changes/periodic-compaction-trigger.md

```diff
@@ -0,0 +1 @@
+* RocksDB now triggers eligible compactions every 12 hours when periodic compaction is configured. This solves a limitation of the compaction trigger mechanism, which only triggers compaction after specific events like flush, compaction, or SetOptions. This mostly affect periodic compaction since files can become eligible for compaction without these events.
```

Comment:
"which only triggers" -> "which would only trigger"

"This mostly affect ..."  As far as we know, it should only affect periodic compaction, right? Can probably remove this sentence. (?)